Terraform crash when using docker_network with aux_address = {foo: ip}

### Terraform Version


0.13.5

### Affected Resource(s)

docker_network

### Terraform Configuration Files
```hcl

resource "docker_network" "vlan" {
  ipam_config {
    aux_address = {
      foo: "1.2.3.4"
   }
  }
}
```

### Panic Output

I can't do that right now.
The config is massive and does leak a lot of important info that I would have to scrub out first.

The bug is easy to reproduce though.

### Expected Behavior

Not crash.

### Actual Behavior

Crash.

### Steps to Reproduce

1. `terraform apply`


### Notes

I'm out of my depth here.

Feels to me though that:

https://github.com/kreuzwerker/terraform-provider-docker/blob/75c8db4e66b3a07007a5c08d6c6f15d50b2dcf7e/docker/resource_docker_network_funcs.go#L127

^ is problematic

I can reproduce the panic.

```hcl
resource "docker_network" "vlan" {
  name = "pr-8"
  ipam_config {
    subnet = "192.168.0.0/16"
    aux_address = {
      foo : "192.168.1.5"
    }
  }
}

provider "docker" {
}

terraform {
  required_providers {
    docker = {
      source = "kreuzwerker/docker"
    }
  }
  required_version = ">= 0.13"
}
```

```
$ TF_LOG=DEBUG terraform apply
```

```
...

panic: interface conversion: interface {} is map[string]string, not map[string]interface {}

...

2020-12-02T12:24:36.815+0900 [DEBUG] plugin.terraform-provider-docker_v2.8.0:   github.com/terraform-providers/terraform-provider-docker/docker/resource_docker_network_funcs.go:231 +0x329

...
```

https://github.com/kreuzwerker/terraform-provider-docker/blob/b8d28d67e632c476e99786805bb6a56603595ba1/docker/resource_docker_network_funcs.go#L231

--- a/docker/resource_docker_network_funcs.go
+++ b/docker/resource_docker_network_funcs.go
@@ -221,7 +221,11 @@ func flattenIpamConfigSpec(in []network.IPAMConfig) *schema.Set { // []interface
 			m["gateway"] = v.Gateway
 		}
 		if len(v.AuxAddress) > 0 {
-			m["aux_address"] = v.AuxAddress
+			aux := make(map[string]interface{}, len(v.AuxAddress))
+			for ka, va := range v.AuxAddress {
+				aux[ka] = va
+			}
+			m["aux_address"] = aux
 		}
 		out[i] = m
 	}

